### ISSUE TITLE
[a11y] Failure in name resolution when validating data set

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1545]
![image](https://user-images.githubusercontent.com/37256050/168083735-1eb24e79-954a-47af-a31f-74dd0d7a18f6.png)

When attempting to validate the WVFlora data set (`/specify/workbench/2/`) on the `nm3` database, I have encountered this error twice

[Specify 7 Crash Report - 2022-05-12T13_18_05.579Z.zip](https://github.com/specify/specify7/files/8679375/Specify.7.Crash.Report.-.2022-05-12T13_18_05.579Z.zip)

**Comment by maxxxxxdlp** [comment]
This is probably specific to the test server.
Does this happen consistently?
If so, you can transfer this issue to the specify7-test-panel repository
